sdk/samples/iot/docs/how_to_iot_hub_samples_windows.md

```diff
@@ -155,13 +155,15 @@ To run the samples, ensure you have the following programs and tools installed o
     PS C:\azure-sdk-for-c\sdk\samples\iot> openssl x509 -noout -fingerprint -in device_ec_cert.pem | % {$_.replace(":", "")} | % {$_.replace("SHA1 Fingerprint=", "")} | Tee-Object fingerprint.txt
     ```
 
-    <details><summary><i>The output will be the fingerprint (also stored in `fingerprint.txt`) and will look similar to:</i></summary>
-    <p?>
+    <details><summary><i>The output will be the fingerprint and will look similar to:</i></summary>
+    <p>
 
     ```powershell
     87B4BAEE5F21CE235A887D703C66FD054AD96701
     ```
 
+    - NOTE: This fingerprint is also stored in `fingerprint.txt`.
```

Comment:
Where is this file?

Reply:
It is generated when the user runs the commands that are in the main README, and lives in the directory that the instructions have the user navigate to in order to run the commands. It should never be checked in.  I imagine it is part of the .gitignore because it would be a hassle to have to remember to delete it after every run of a certificate sample while making changes to check in, and most likely one would forget to delete it.

Reply:
I'll update it to say the generated file